modules/swagger-codegen/src/main/java/io/swagger/codegen/v3/config/CodegenConfigurator.java

```diff
@@ -674,6 +678,21 @@ private ParseOptions buildParseOptions() {
         options.setFlatten(true);
         options.setFlattenComposedSchemas(flattenInlineSchema);
         options.setSkipMatches(this.skipInlineModelMatches);
+
+        if (Objects.equals(System.getenv("SAFELY_RESOLVE_URL"), "true")) {
```

Comment:
would apply the same logic, limited to the new options, to https://github.com/swagger-api/swagger-codegen/pull/12140/files#diff-98b18ce4b3c3949e219dcc49c873cae48091f65f5be0e55d299617d03588dbe1L581

Reply:
Yes, let's keep consistency using same logic

Reply:
Thanks for the comment! :) Sorry about the delay, I was unavailable last couple of days.